src/frontend/src/pages/CreditsPage/CreditsPage.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { Grid, Box, Button } from '@mui/material';
+import PageTitle from '../../layouts/PageTitle/PageTitle';
+import PageBlock from '../../layouts/PageBlock';
+import { useState } from 'react';
+
+const CreditsPage: React.FC = () => {
+  const names = ['Kevin Chen']; // This is a list of names, add yours here!
+
+  const [displaySnark, SetDisplaySnark] = useState(false);
+
+  return (
+    <PageBlock>
+      <PageTitle title={'CREDITS'} previousPages={[]} />
+      <Grid container spacing={2}>
+        {names?.map((name) => (
+          <Grid item>
+            <Box>{name}</Box>
```

Comment:
```suggestion
            <Typography>{name}</Typography>
```

No need for the unnecessary additional nested layout components (grid and box). Instead, since its text, format it properly by wrapping it in typography. You'll need to add the specification of what variant it should be

Reply:
got it.